### PR TITLE
LPS-126361

### PR DESCRIPTION
**Pull request by alexcandidos**

### CODE REVIEW

**Comment by Tim-Cao** [comment]
Hi @alexcandidos, this case is already covered by LocalizedNestedFieldsWebContent#TranslateWebContentToNLanguages

**Comment by emanuelfisilva** [comment]
Thanks for your review @Tim-Cao , but we'll choose to keep these validations separated.

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedNestedFieldsWebContent.testcase [hunk 1]
```diff
@@ -602,12 +602,106 @@ definition {
...
+		WebContentStructures.saveCP();
+
+		Navigator.gotoNavTab(navTab = "Web Content");
+
+		WebContentNavigator.gotoAddWithStructureCP(structureName = "Test Structure");
+
+		WebContent.addWithStructureCP(
+			structureName = "Test Structure",
+			webContentTitle = "WC English");
+
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedNestedFieldsWebContent.testcase:649]: Use existing macro

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3780
View total diff https://github.com/liferay-echo/liferay-portal/pull/3780/commits/c6b7903bfd651080ded2ab1ab65a3c4fae1c35dc

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**